### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by leeyr338**
add README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Provide the type of interaction between the cita-node 
+interaction type and the cita component, use protobuf 
+to generate the corresponding file. And have generated
+files for the following languages in advance:
+- Rust
+- go
+- javascript
+- java
+
+You can use them derectly, and be free to use the `protos`
```
Comment (yejiayu) [inline README.md]: `s/protos/protoc`

Comment (leeyr338) [reply README.md]: thanks! and  I mean "use the photo files in protos directory."

**Review by yejiayu — COMMENTED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Provide the type of interaction between the cita-node 
+interaction type and the cita component, use protobuf 
+to generate the corresponding file. And have generated
+files for the following languages in advance:
+- Rust
+- go
+- javascript
+- java
+
+You can use them derectly, and be free to use the proto
```
Comment (u2) [inline README.md]: directly

Comment (leeyr338) [reply README.md]: ok

**Review by u2 — APPROVED** [review]

**Pull request merged**